modules/tinymce/src/core/main/ts/html/ParserFilters.ts

```diff
@@ -184,21 +155,24 @@ const register = (parser: DomParser, settings: DomParserSettings): void => {
 
   registerBase64ImageFilter(parser, settings);
 
+  const shouldSandboxIframes = settings.sandbox_iframes ?? false;
+  const sandboxIframesExclusions = settings.sandbox_iframes_exclusions ?? {};
```

Comment:
So this would be the spot where I would convert the array of items into a `Set` and then use that to check for host names.